x-pack/plugins/triggers_actions_ui/public/application/components/builtin_alert_types/threshold/expression.scss

```diff
@@ -1,3 +1,7 @@
 .actAlertVisualization__chart {
   height: $euiSize * 15;
 }
+
+.ofAggFieldContainer {
+  width: 480px;
+}
```

Comment:
I don't think this is the right file to add this selector to. Each JS file needs to have a complimentary SASS file next to it that gets imported via the main plugin's `index.scss` file.